packages/headless/src/api/search/date/date-format.test.ts

```diff
@@ -18,8 +18,8 @@ describe('#isSearchApiDate', () => {
 describe('#formatDateForSearchApi', () => {
   it('returns the correct format', () => {
     const date = 818035920000;
-    expect(formatDateForSearchApi(dayjs(date))).toBe(
-      dayjs(date).format('YYYY/MM/DD@HH:mm:ss')
+    expect(formatDateForSearchApi(parseDate(date))).toBe(
+      parseDate(date).format('YYYY/MM/DD@HH:mm:ss')
```

Comment:
There should be the inverse test: What happens when you pass in a date format that is not the API format.